### PR TITLE
En 1144 activity stream

### PR DESCRIPTION
**Pull request by stefanopuddu**
Added activityStream endpoint
Moved social stuff from admin-console to core

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: engine/src/main/java/org/entando/entando/aps/system/services/activitystream/ActivityStreamService.java [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    private static final String KEY_FILTER_PARAMS = "params";
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    private ActionLogRecordDtoBuilder dtoBuilder = new ActionLogRecordDtoBuilder();
+
+    @Autowired
+    private IActionLogManager actionLogManager;
+
+    private ISocialActivityStreamManager socialActivityStreamManager;
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/aps/system/services/activitystream/ActivityStreamService.java]: Does this need to get @Autowired?

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Pull request merged**